server/src/lib/basket.ts

```diff
@@ -38,11 +40,39 @@ function toISO(date: string) {
   return date ? new Date(date).toISOString().slice(0, -5) + 'Z' : null;
 }
 
+/*
+ * Wrapper function to sync user activity to email provider
+ * - Basket = MoCo internal
+ * - SQS = used by MoFo agency Cinchy to talk to Acoustic
+ */
+async function syncToEmailProvider(data: any) {
+  if (getConfig().CINCHY_ENABLED) {
+    const cinchyParams = {
+      MessageBody: JSON.stringify(data),
+      QueueUrl: getConfig().CINCHY_CONFIG.endpoint
+    }
+
+    sqs.sendMessage(cinchyParams, (err, data) => {
+      if (err) {
+        console.log("Cinchy SQS error:", err);
+      }
+    });
+  } else {
+    sendRequest({
+      ...basketConfig,
+      form: data,
+    }).catch(err => console.error(err.message));
+  }
+}
+
+/*
+ * Fetch the most recent time the user was synced
+ */
 async function getCurrentStatus(client_id: string) {
   const [[row]] = await db.query(
     `SELECT u.*,
       NOW() AS new_last_active,
-      HOUR(TIMEDIFF(NOW(), u.last_active)) AS hours_elapsed
+      MINUTE(TIMEDIFF(NOW(), u.last_active)) AS mins_elapsed
```

Comment:
Is there a reason we're changing the sync time from 1 hr to 5 min? Do we know approximately the volume of events being currently sent?

Reply:
5 minutes sounds good for starters! R+